docker/dev/docker-compose.yml

```diff
@@ -49,8 +50,10 @@ services:
     restart: always
 
   pictrs:
-    image: asonix/pictrs:0.3.0-beta.11
+    image: asonix/pictrs:0.3.0-beta.12-r1
```

Comment:
This was missing from dockerhub, it'll need to be pushed or downgraded before we merge to main.

Reply:
Oh oops, I accidentally tagged it 3.0.0 instead of 0.3.0 :/